package.json

```diff
@@ -47,7 +47,7 @@
         "live-server": "^1.2.1",
         "local-web-server": "^2.6.0",
         "mocha": "^4.1.0",
-        "prettier": "1.11.1",
+        "prettier": "^1.17.0",
         "sinon": "^6.0.1",
```

Comment:
Specifically with Prettier, I think it's worth locking the version so we don't get diffs by accidentally updating on a new install.